sys/include/net/netopt.h

```diff
@@ -248,6 +248,11 @@ typedef enum {
      * @note    Interfaces are not meant to respond to that.
      */
     NETOPT_NUMOF,
+
+    /**
+     * @brief   add/remove an address to the link layer filter list
```

Comment:
- "remove" should be removed here as it needs an other netopt parameter. Addition and removal are not distinguished here
- expects a pointer to `l2filter_t struct` on`get` or to a hardware address (usually `uint8_t`) on `set`